godot/entities/characters/player/player.gd

```diff
@@ -3,59 +3,40 @@ extends CharacterBody2D
 ## Velocidad de movimiento del personaje.
 @export_range(10, 1000, 10) var SPEED = 300.0
 
-## Velocidad de movimiento al correr del personaje.
-@export_range(10, 1000, 10) var RUN_SPEED = 500.0
```

Comment:
Aquí vemos que este pull request borra cosas que están en main. Por ejemplo estas líneas se agregaron en https://github.com/GameNova-Studio/path-to-the-throne/pull/140